rpcserver.go

```diff
@@ -379,8 +379,24 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 				update.Error)
 		}
 
+		batches, err := r.cfg.AssetMinter.ListBatches(update.BatchKey)
+		if err != nil {
+			return nil, fmt.Errorf("unable to list batches: %w",
+				err)
+		}
+
+		rpcBatches, err := fn.MapErr(batches, marshalMintingBatch)
```

Comment:
Would be useful to have a `--short` option that omits the `Assets` array, for large batches (or some other form of shorter batch response?).

Reply:
Good idea, done.